mako.1.scd

```diff
@@ -183,96 +183,15 @@ dismissed with a click or via *makoctl*(1).
 
 	Default: none
 
-# CRITERIA-ONLY STYLE OPTIONS
+# CONFIGURATION FILE
 
-Some style options are not useful in the global context and therefore have no
-associated command-line option.
+The mako configuration file is located at *~/.config/mako/config* or at
+*$XDG\_CONFIG\_HOME/mako/config*.
```

Comment:
I would leave this out since it is also present in mako.5, so that we don't end up forgetting to update one or the other at some point down the road.